mm2src/common/write_safe/io.rs

```diff
@@ -17,14 +15,10 @@ macro_rules! writeln_safe_io {
     }};
 }
 
-pub trait WriteSafeIO {
-    fn write_safe<'a>(&mut self, args: fmt::Arguments<'_>)
-    where
-        Self: DerefMut<Target = &'a mut dyn Write>,
-    {
-        let writer = self.deref_mut();
-        Write::write_fmt(writer, args).expect("`write_fmt` should never fail for `WriteSafeIO` types")
+pub trait WriteSafeIO: std::io::Write {
```

Comment:
No need to use the fully qualified import for `Write` since it's already imported.

Reply:
Thank you, [got rid of it](https://github.com/KomodoPlatform/komodo-defi-framework/pull/1932/commits/4542946231d0571097329e40a3671e44b19ce248)